tests/test_bgp_neighbor.rb

```diff
@@ -483,10 +483,11 @@ def test_transport_passive_mode
       neighbor = create_neighbor(vrf)
       check = []
       if platform == :ios_xr
-        check = [:active_only, :passive_only, :both,
+        check = [:active_only, :passive_only, :both, :none,
```

Comment:
:both is still valid for XR.  The default has been changed to :none (which will clear the setting).